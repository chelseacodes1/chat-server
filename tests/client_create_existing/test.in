REGISTER user29 brazil 
LOGIN user29 brazil 
CHANNELS
CREATE news
JOIN news
JOIN intros
JOIN f1 