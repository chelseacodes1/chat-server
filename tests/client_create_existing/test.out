RESULT REGISTER 1
RESULT LOGIN 1
RESULT CHANNELS f1, intros, local, news, 茶馆
RESULT CREATE news 0
RESULT JOIN news 1
RESULT JOIN intros 1
RESULT JOIN f1 1
